Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -309,54 +309,62 @@ theorem nthRoots_two_eq_zero_iff {r : R} : nthRoots 2 r = 0 ↔ ¬IsSquare r :=
   simp_rw [isSquare_iff_exists_sq, eq_zero_iff_forall_not_mem, mem_nthRoots (by norm_num : 0 < 2),
     ← not_exists, eq_comm]
 
-/-- The multiset `nthRoots ↑n (1 : R)` as a Finset. -/
-def nthRootsFinset (n : ℕ) (R : Type*) [CommRing R] [IsDomain R] : Finset R :=
+/-- The multiset `nthRoots ↑n a` as a Finset. -/
+def nthRootsFinset (n : ℕ) {R : Type*} (a : R) [CommRing R] [IsDomain R] : Finset R :=
```

Comment:
I agree that we should change this, but I think that we should note in the new docstring that this definition used to mean something else to help porting downstream projects that use it.

Reply:
Thanks. I've updated the docstring.